.github/workflows/run_shark_tank.yml

```diff
@@ -7,7 +7,7 @@ on:
   # Uncomment `pull_request` below to test your PR on SHARK.
   # Results will be uploaded to gs://shark-benchmark-artifacts.
   # See the workflow's `SHARK -> generate_report` log output for exact results URL.
-  # pull_request:
+  pull_request:
```

Comment:
I found that this workflow doesn't have concurrency control so every push in PR will create a new job in the queue and not cancel the previous one.

I'm sending a fix. In the meantime, could you add
```suggestion
  pull_request:
  
concurrency:
# A PR number if a pull request and otherwise the commit hash. This cancels
# queued and in-progress runs for the same PR (presubmit) or commit
# (postsubmit).
  group: ${{ github.event.number || github.sha }}
  cancel-in-progress: true
```